doc_source/glue-best-practices.md

```diff
@@ -263,4 +263,4 @@ To reduce the likelihood that Athena is unable to read the `SMALLINT` and `TINYI
 
 ### Automating AWS Glue jobs for ETL<a name="schema-etl-automate"></a>
 
-You can configure AWS Glue ETL jobs to run automatically based on triggers\. This feature is ideal when data from outside AWS is being pushed to an Amazon S3 bucket in a suboptimal format for querying in Athena\. For more information, see [Triggering AWS Glue jobs](https://docs.aws.amazon.com/glue/latest/dg/trigger-job.html) in the *AWS Glue Developer Guide*\.
\ No newline at end of file
+You can configure AWS Glue ETL jobs to run automatically based on triggers\. This feature is ideal when data from outside AWS is being pushed to an Amazon S3 bucket in a suboptimal format for querying in Athena\. For more information, see [Triggering AWS Glue jobs](https://docs.aws.amazon.com/glue/latest/dg/trigger-job.html) in the *AWS Glue Developer Guide*\.
```

Comment:
don't believe i've changed anything here 🤔 Little github funkyness?

Reply:
No worries, thanks for the link fixes!